pontoon/base/views.py

```diff
@@ -456,6 +457,82 @@ def get_team_comments(request):
     return JsonResponse(payload, safe=False)
 
 
+def _send_add_comment_notifications(user, comment, entity, locale, translation):
+    # On translation comment, notify:
+    #   - authors of other translation comments in the thread
+    #   - translation author
+    #   - translation reviewers
+    if translation:
+        recipients = list(translation.comments.values_list("author__pk", flat=True))
```

Comment:
Should `recipients` be a `set` instead of a list? There's no point sending a notification to the same user several times.